runtime/shibuya/src/lib.rs

```diff
@@ -528,8 +528,17 @@ impl OnUnbalanced<NegativeImbalance> for DealWithFees {
             if let Some(tips) = fees_then_tips.next() {
                 tips.merge_into(&mut fees);
             }
+            let (to_burn, collators) = fees.ration(20, 80);
+
+            // burn part of fees
+            let burned = Balances::burn(to_burn.peek());
+            assert!(
```

Comment:
can you use error instead of assert?

Reply:
Unfortunately we can’t, but we can escape of using assert here